Assets/Scripts/Board/Graph.cs

```diff
@@ -12,13 +13,25 @@ public enum Building
         City,
     }
 
+    public enum VertexDirection
+    {
+        North,
+        Northeast,
+        Southeast,
+        South,
+        Southwest,
+        Northwest,
+    }
+
```

Comment:
It feels weird to have one of these in Graph and one in Board. they're different directions due to literal geometry, but still imagine it being cleaner to put the enums in the same class, not sure which

Reply:
Moved to Graph.